_data/engine-cli/docker_secret_create.yaml

```diff
@@ -1,8 +1,8 @@
 command: docker secret create
 short: Create a secret from a file or STDIN as content
-long: |-
-  Creates a secret using standard input or from a file for the secret content. You must run this
-  command on a manager node.
+long: "Creates a secret using standard input or from a file for the secret content.
```

Comment:
@tiborvass I don't see anything in the source file that is causing this. Why is the YAML generator not putting the  `|-`?